index.js

```diff
@@ -31,6 +37,13 @@ function init(editor) {
 			}
 		}
 
+		if (config.end_of_line && lineEndings.hasOwnProperty(config.end_of_line)) {
+			var preferredLineEnding = lineEndings[config.end_of_line];
+			var buffer = editor.getBuffer();
+			buffer.setPreferredLineEnding(preferredLineEnding);
```

Comment:
The atom/line-ending-selector plugin pretty much does the same thing, so not sure they've implemented it the way you'd prefer.